sound/soc/sof/topology.c

```diff
@@ -1919,6 +1919,8 @@ static int sof_manifest(struct snd_soc_component *scomp, int index,
 /* vendor specific kcontrol handlers available for binding */
 static const struct snd_soc_tplg_kcontrol_ops sof_io_ops[] = {
 	{SOF_TPLG_KCTL_VOL_ID, snd_sof_volume_get, snd_sof_volume_put},
+	{SND_SOC_TPLG_CTL_VOLSW, snd_sof_volume_get, snd_sof_volume_put,
```

Comment:
Still not following why we need to override a standard kcontrol ID here.

Reply:
I also don't get what this is about?

Reply:
@plbossart  @lgirdwood  please refer to https://github.com/thesofproject/soft/pull/34